airlock/business_logic.py

```diff
@@ -124,16 +124,17 @@ class CommentVisibility(Enum):
     @classmethod
     def choices(cls):
         return {
-            CommentVisibility.PRIVATE: "Only visible to output-checkers",
-            CommentVisibility.PUBLIC: "Visible to all",
+            Visibility.PRIVATE: "Only visible to output-checkers",
+            Visibility.PUBLIC: "Visible to all",
         }
 
     def description(self):
         return self.choices()[self]
 
-    @cached_property
     def independent_description(self):
-        return "Only visible to you until both reviews completed"
+        return (
+            self.description() + " (Only visible to you until both reviews completed)"
```

Comment:
I think this wording needs some thought. It makes the visibility pill very long, and it always stays visible, so at smaller screen sizes it covers up the username. Would " (pending)"  be enough, if users have an understanding already of how the independent review process works? Or some sort of icon?  

![image](https://github.com/user-attachments/assets/e09e4af4-730e-400f-b803-8acc71dd489f)

Reply:
Yeah, I wasn't happy with this, but I didn't try reducing width, which is clearly not going to work.

I've used Pending is to mean "won't be visible until the turn is complete", and Hidden to mean "hidden from other output checkers until 2 review complete", aka blinded.

Maybe the way forward here is to have pill that says either "Private" or "Public", and another pill that says "Hidden" (or perhaps "Blinded"), but with tooltips to have enough space to clearly explain what we mean, rather than trying to cram the text in a pill?

Reply:
Yeah, I think one(ish) word pills with tooltips would be better

Reply:
Given we haven't got tooltips sorted yet, I'll change this to one word, and add ticket for adding tooltips